embedchain/embedchain.py

```diff
@@ -500,13 +500,18 @@ def retrieve_from_database(self, input_query: str, config: Optional[BaseLlmConfi
 
             db_query = ClipProcessor.get_text_features(query=input_query)
 
-        contents = self.db.query(
+        data = self.db.query(
             input_query=db_query,
             n_results=query_config.number_documents,
             where=where,
             skip_embedding=(hasattr(config, "query_type") and config.query_type == "Images"),
         )
-        return contents
+
+        contexts = data
```

Comment:
You probably dont need two variables, right? 

Can just change `data` variable name to `contexts` and use that throughout?

Reply:
sure